source/s3_util.c

```diff
@@ -62,9 +62,11 @@ const struct aws_byte_cursor g_user_agent_header_product_name =
     AWS_BYTE_CUR_INIT_FROM_STRING_LITERAL("CRTS3NativeClient");
 
 const struct aws_byte_cursor g_s3_internal_error_code = AWS_BYTE_CUR_INIT_FROM_STRING_LITERAL("InternalError");
-const struct aws_byte_cursor g_s3_slow_down_error_code = AWS_BYTE_CUR_INIT_FROM_STRING_LITERAL("SlowDown");
+
+const struct aws_byte_cursor s_s3_slow_down_error_code = AWS_BYTE_CUR_INIT_FROM_STRING_LITERAL("SlowDown");
+const struct aws_byte_cursor s_s3_time_too_skewed = AWS_BYTE_CUR_INIT_FROM_STRING_LITERAL("RequestTimeTooSkewed");
 /* The special error code as Asynchronous Error Codes */
-const struct aws_byte_cursor g_s3_internal_errors_code = AWS_BYTE_CUR_INIT_FROM_STRING_LITERAL("InternalErrors");
+const struct aws_byte_cursor s_s3_internal_errors_code = AWS_BYTE_CUR_INIT_FROM_STRING_LITERAL("InternalErrors");
```

Comment:
trivial: can we put "InternalErrors" next to "InternalError"?

I realize they were already separated, but this change makes them even further apart

or just don't even have these variables at the top of the file, just type `"InternalErrors"` in the 1 place it's actually used in code